### ISSUE TITLE
508-defect-2 [COGNITION]: Content structure MUST be organized clearly, and follow the Plain Language Guidelines

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#16203]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** Jennifer

## User Story or Problem Statement

As a user with cognitive considerations, I need the information in an organized, clear, and simple presentation so that I may understand the content.

## Details

The presentation of the supporting information for the "Don't see the issue you're looking for?" additional info component **must** meet the requirements of the Plain Writing Act and follow the Plain Language Guidelines. The current content design has "[icon] download and fill out VA Form 20-0996 PDF (1.5MB)" within the bulleted list content. This makes it distracting and difficult to digest. It would be improved if the instructions were separated from the download link (see recommended solution).

## Acceptance Criteria

- [ ] Download links must be concise and not contain extra instructions
- [ ] Download link is presented on its own line 

## Environment

* Operating System: all
* Browser: all
* Server destination: staging

## Steps to Recreate

1. Enter `https://staging.va.gov/decision-reviews/higher-level-review/request-higher-level-review-form-20-0996/contested-issues` in browser
2. Navigate to Step 2 of 4: Contested issues
3. Verify that the download links presented within the bulleted content is difficult to digest

### Current 

* We made the decision over a year ago. You have 1 year from the date on your decision letter to request a Higher-Level Review. Note: If you weren’t able to request a timely review due to COVID-19 pandemic, we may grant you an extension of the deadline. To request an extension, [[icon] download and fill out VA Form 20-0996 PDF (1.5MB)](https://www.vba.va.gov/pubs/forms/VBA-20-0996-ARE.pdf) and attach a note that you’re requesting a filing extension due to COVID-19.

### Recommendation 

* We made the decision over a year ago. You have 1 year from the date on your decision letter to request a Higher-Level Review. Note: If you weren’t able to request a timely review due to COVID-19 pandemic, we may grant you an extension of the deadline. To request an extension, download and fill out VA Form 20-0996 and attach a note that you’re requesting a filing extension due to COVID-19.
[[icon] Download VA Form 20-0996 PDF (1.5MB)](https://www.vba.va.gov/pubs/forms/VBA-20-0996-ARE.pdf)

## WCAG or Vendor Guidance (optional)

* [Plainlanguage.gov - Organize the information](https://www.plainlanguage.gov/guidelines/organize/)

### Current

![Screen Shot 2020-11-16 at 10 50 00 AM](https://user-images.githubusercontent.com/57469/99275445-7f3d5d00-27f9-11eb-924e-47ce321105d0.png)

### Recommendation

![Screen Shot 2020-11-16 at 11 06 46 AM](https://user-images.githubusercontent.com/57469/99277493-dc3a1280-27fb-11eb-8618-d8414e0deef4.png)

**Comment by Mottie** [comment]
The content of this dropdown is changing based on the recommendations in this issue - https://github.com/department-of-veterans-affairs/va.gov-team/issues/16211, once the changes have been finalized, we'll tag you to review. Thanks!

**Comment by jenstrickland** [comment]
Thanks for the update, @Mottie ! Please share the design recommendations with the team as they consider how to implement the changes. The gist of it is, streamline the data presentation to be empathetic for those with cognitive considerations and screen reader users. The link repeated throughout is confusing, for cognitive considerations and for those navigating by tab.

**Comment by Mottie** [comment]
These content updates have been merged and should be available in staging soon @joshkimux

**Comment by Mottie** [comment]
More updates incoming. See https://github.com/department-of-veterans-affairs/va.gov-team/issues/18065

**Comment by joshkimux** [comment]
@Mottie Note here just it isn't too clear in the ticket-- including links repetitively in each of the `li` items will make for a better experience for screen reader users as action items will be provided immediately in context!

**Comment by Mottie** [comment]
We ended up with just one download link at the bottom, if that's what you mean?

<details><summary>Expanded list</summary>


![Screen Shot 2021-01-05 at 11 11 13 AM](https://user-images.githubusercontent.com/136959/103676532-cff24c00-4f46-11eb-8f99-d051934e6283.png)</details>

My previous comment is referring to content changes that probably aren't related to the links.

**Comment by joshkimux** [comment]
@Mottie Thank you for clarifying! And yes, that's exactly what I was referring to. The main concern is that users may not know that the download link will exist at the end of the list or that they may struggle remembering what they need and would have to renavigate back into/out of the list. By providing the necessary links/downloads within each list item, they'll be able to access it within the context of the content itself.

**Comment by kevinstachura** [comment]
@joshkimux @Mottie in a previous version, we did have the links in each and every `li` item that specified the PDF should be the action, but removed based upon later content changes to the design. @luke-gcio was this a content change that was based solely upon client request? Just want to confirm we had spoken to the right parties (Peggy, Jennifer, etc) to ensure compliance on this item 👍👍 thanks!

**Comment by joshkimux** [comment]
@peggygannon See above!

**Comment by peggygannon** [comment]
Hi, @joshkimux Yes, I had recommended moving the PDF link to the bottom of the alert in an effort to streamline the content and to not have the PDF link repeated several times in the alert box, (taking into account mobile).

Above, Jen S had provided earlier accessibility feedback that repeating the link could be confusing for those with cognitive issues. Please let me know if I'm misreading this earlier accessibility feedback? Happy to adjust the content as needed. Let me know. 

cc: @luke-gcio and @kevinstachura

**Comment by joshkimux** [comment]
@peggygannon thank you for the additional context! Gotcha, I think the unanswered question then in this thread is around where that first download link was intended to be. Jennifer's original screenshot had it on the first bullet, so I think her original intent was to have it emerge within the list sooner rather than later so users would know where to look for it.

That being said, I don't think this is worth stressing over if that's the case and she was comfortable with just one link + having the advantages of streamlining on mobile.

Closing 🙇
